source/minio-cli/minio-kes-reference.rst

```diff
@@ -0,0 +1,421 @@
+======================================
+MinIO Key Encryption Service (``kes``)
```

Comment:
@aead this is still a little rough but I'd like to get your feedback on just this page from a correctness perspective.

ToDo on this page is to describe how KES handles cryptographic functions (Something we can reference any algorithms we're using to compute keys and such. Good for companies that want to scan to see whether we're using compliant algos and such.)